Jenkinsfile

```diff
@@ -22,6 +22,23 @@ pipeline {
                 sh ".github/cicd/player.sh build fig-validation-service dev"
             }
         }
+        stage('SCM') {
+            git 'gttps://github.com/bcgov/jag-dps'
```

Comment:
is gttps a new protocol ?

Reply:
gypertext transfer protocol secure....didn't you get the memo.

Reply:
oh, got it now

Reply:
fixed.